WordPress/Classes/ViewRelated/Stats/Double+Stats.swift

```diff
@@ -0,0 +1,32 @@
+import Foundation
+
+extension Double {
+
+    func abbreviatedString() -> String {
```

Comment:
This is screaming for some comments explaining what this method _actually_ does ;P 

Some explanation of black magic (I have _no idea_ what's going on with that `log10` down there and why it works) would be nice too, but I'll settle for just header docs showing what are example inputs/outputs of this.

Reply:
Haha! Ya know, I totally forgot about comments. You of course are correct. I'll add some!